lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php

```diff
@@ -123,41 +130,76 @@ public function convertException($message, DriverException $exception)
 
     /**
      * {@inheritdoc}
+     *
+     * @return AbstractPlatform|MariaDb102Platform|MySQL57Platform|MySqlPlatform
```

Comment:
Doesn't `AbstractPlatform` cover everything?

Reply:
Good catch, remaining of my attempts with blob defaut support. Thanks